shellbase-slack/src/main/scala/com/sumologic/shellbase/slack/PostCommandToSlack.scala

```diff
@@ -42,15 +43,19 @@ trait PostCommandToSlack extends ShellCommand with PostToSlackHelper {
     }
   }
 
-  protected def slackMessage(commandPath: List[String], arguments: List[String]): Option[String] = {
+  // VisibleForTesting
+  protected[slack] def slackMessage(commandPath: List[String], arguments: List[String],
```

Comment:
I'm not sure if the package change is a breaking change.  Interesting.  (The added arg is though.)